components/raftstore/src/store/peer.rs

```diff
@@ -2593,9 +2599,9 @@ impl Peer {
         }
     }
 
-    pub fn bcast_wake_up_message<T: Transport>(&self, trans: &mut T) {
-        let region = self.raft_group.store().region();
-        for peer in region.get_peers() {
+    pub fn bcast_wake_up_message<T: Transport, C>(&self, ctx: &mut PollContext<T, C>) {
+        ctx.need_flush_trans = true;
```

Comment:
What if peers is empty?

Reply:
It's another case that can not be handled by this PR.